docs/webhook.asciidoc

```diff
@@ -7,7 +7,7 @@ A validating webhook provides additional validation of Elasticsearch resources:
 === Architecture
 The webhook is composed of 4 main components. Here is a brief description of each of them to understand how they interact, their naming, and how they are managed.
 
-. A `ValidatingWebhookConfiguration` object that defines the validating webhook, named `elastic-es-validation.k8s.elastic.co`. It must be created before starting the operator. The `caBundle` field can be automatically managed as part of the automatic certificate management _(see below)_.
+. A `ValidatingWebhookConfiguration` object that defines the validating webhook, named `elastic-es-validation-{eck_crd_version}.k8s.elastic.co`. It must be created before starting the operator. The `caBundle` field can be automatically managed as part of the automatic certificate management _(see below)_.
```

Comment:
The `ValidatingWebhookConfiguration` object is named `elastic-webhook.k8s.elastic.co`
It is the `clientConfig` that is named `elastic-es-validation-{eck_crd_version}.k8s.elastic.co`

Reply:
The `ValidatingWebhookConfiguration` contains a `webhooks` array of `Webhook`, for which we define the name. That's not the name of the `clientConfig`.

I think in "A `ValidatingWebhookConfiguration` object that defines the validating webhook, named `elastic-es-validation-{eck_crd_version}.k8s.elastic.co`",  `named` refers to the inner validating webhook, not to the ValidatingWebhookConfiguration.

WDYT?

Reply:
We can maybe simplify that sentence to `A ValidatingWebhookConfiguration` object that defines the validating webhooks`(without any name reference) so we avoid the confusion?